summarize.go

```diff
@@ -0,0 +1,21 @@
+package cohere
+
+type SummarizeOptions struct {
```

Comment:
where is `extractiveness`? also the comments seem to be less explanatory compared to node's for example. can you sync them please?

Reply:
Good catch. updated